doc/Language/containers.pod6

```diff
@@ -383,6 +371,7 @@ declaration:
     my Int:D $def = 3;
     say $def;   # OUTPUT: «3␤»
     $def = Int; # Typecheck failure
+=comment ^^^ fails at runtime, so xt/example-compilation still passes
```

Comment:
Not terribly fond of these comments. I would really prefer to open an issue, to be closed by whoever.